### DISCUSSION TITLE
Conteúdo anexo de uma talk de um evento.

### DISCUSSION THREAD

**Discussion by abpaula77** [dev-pira/site#230] [Ideas]
A cada talk dentro de um evento que é realizado pela comunidade existe a possibilidade do palestrante disponibilizar o conteúdo apresentado na talk.
Esse conteúdo pode ser um ppt, um codigo fonte, uma pesquisa ou links paraum reposítório ou ferramentas e sites citados.
Poderia ter no card da talk  a possibilidade de cadastrar os links para o conteúdo e até eventualmente a possíbilidade de armazenar a informação no próprio site. (com limites).
Exemplos recentes: material enviado pelo Balta da palestra no github (colocar o link)
programas citados pelo Murilo que foi colocado o link no grupo.

**Comment by murilobeltrame** [comment]
Boa Ale.
Achei massa a ideia.
Pelo que conversamos, vale colocar links: seja de GitHub, Drive, ou outra coisa ...
Fiz uma menção na task #68 , pode valer a pena os conteúdos estarem la tb.
